### PR TITLE
OSOE-91: Normalize line endings to auto across all submodules

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-91](https://lombiq.atlassian.net/browse/OSOE-91)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (0liver) [inline .gitattributes:1]: Why did you remove the _.placeholder_ file? (Can't comment in that one directly.)

**Review by 0liver — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (sarahelsaig) [reply .gitattributes:1]: See https://github.com/Lombiq/Orchard-UIKit/pull/13#discussion_r856669196

**Review by 0liver — APPROVED** [review]

**Review by 0liver — APPROVED** [review]
